VirtoCommerce.PricingModule.Web/Scripts/blades/assignment-list.js

```diff
@@ -140,6 +167,17 @@ function ($scope, assignments, dialogService, uiGridHelper, bladeUtils, catalogs
         bladeUtils.initializePagination($scope);
     };
 
+    function searchAssignments(skip, take, onDone) {
```

Comment:
This method could be inlined back to blade.refresh.

Reply:
Done